code/game/objects/items/mail.dm

```diff
@@ -332,3 +334,166 @@
 /obj/item/paper/fluff/junkmail_generic/Initialize(mapload)
 	default_raw_text = pick(GLOB.junkmail_messages)
 	return ..()
+
+// =====================
+// Antag stuff down here
+// =====================
+
+/obj/item/mail/traitor
+	var/armed = FALSE
+	var/mob/madeby
```

Comment:
This is going to hard delete when the mob does. It looks like this is only used for admins, so replace this with variables for the ckey and their current name instead.

Reply:
I am already on it. I will cache thier ckey and mind name.